wait-task/README.md

```diff
@@ -0,0 +1,56 @@
+# Wait Custom Task for Tekton
+
+This repo provides an experimental [Tekton Custom
+Task](https://tekton.dev/docs/pipelines/runs/) that, when run, simply waits a
+given amount of time before succeeding, specified by an input parameter named
+`duration`.
+
+### Motivation
+
+Some users might want their Pipeline to wait some amount of time between two
+tasks, for example, to deploy to one cluster then wait an hour before deploying
+to other clusters.
```

Comment:
it may be helpful to provide an example pipelinerun with this scenario

Reply:
Excellent idea! Done.